src/main.rs

```diff
@@ -0,0 +1,11 @@
+use sha3::{Digest, Sha3_256};
+
+fn main() {
+    let mut hasher = Sha3_256::new();
+    hasher.update(b"1");
+    let hash = hasher.finalize();
```

Comment:
For a single value you can do
`Sha3_256::digest(b"1")`

But it's ok don't change it here because it's just test code